README.md

```diff
@@ -25,13 +25,39 @@ Or install it yourself as:
 ```ruby
 require 'zeus/api_client'
 zeus_client = Zeus::APIClient.new({
-    :access_token => "your_token_here",
+    :token => "your_token_here",
```

Comment:
@victorfonsec4 
I'm not sure, but I guess it was 2.1.0 before.
Since I am doing this refactoring in version 2.4.1, I updated Gemfile and travis setting to 2.4.1.
And you are right! For this newer version it's `token: "your token here"`. Just updated the README about it!